CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 3.6.1
 
 - use `either` compatibility library instead of shims
+- make `CCSeq.to_array` only traverse the sequence once
```

Comment:
that's in 3.7, but don't worry about updating the changelog, I do it at release time.

Reply:
I reverted this bit.